LICENSE

```diff
@@ -1,176 +1,15 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+https://spdx.org/licenses/BSD-2-Clause-Patent.html
 
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
 
-   1. Definitions.
+1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
```

Comment:
Well, it explicitly mentions "the above copyright" that was removed. It sounds like we need to add it back then.

Reply:
Updated the LICENSE file, please take one more look see if it looks better.